src/middlewares/messageValidation.js

```diff
@@ -0,0 +1,31 @@
+import Joi from '@hapi/joi';
+
+export default async (req, res, next) => {
+  const schema = Joi.object({
+    names: Joi.string()
+      .required()
+      .trim()
+      .regex(/^[a-z]{2}([a-z][\W]*)/i)
+      .messages({
+        'string.empty': 'Your names are required',
+        'string.pattern.base': 'Names should be valid names'
+      }),
+    email: Joi.string().email().required().trim()
+      .lowercase()
+      .messages({
+        'string.email': 'Email must be a valid email',
+        'string.lowercase': 'Email should be in lowercase',
```

Comment:
This should not be shown as an error, you just have to lowercase every email entered by a user.